vllm/compilation/pass_manager.py

```diff
@@ -53,6 +64,9 @@ def add(self, pass_: InductorPass):
         assert isinstance(pass_, InductorPass)
         self.passes.append(pass_)
 
+    def uuid(self):
+        return self.__getstate__()
+
```

Comment:
Actually, it failed on the unpickling of this class...

Reply:
I assume this is related to [this issue](https://github.com/vllm-project/vllm/issues/15223)?